core/src/main/resources/lib/layout/breadcrumbs.css

```diff
@@ -24,13 +24,16 @@
     margin: 0;
     height: 2em;
     border-bottom: 1px solid #d3d7cf;
+    display:block;
 }
 
 #breadcrumbs LI {
     float:left;
     line-height:2em;
     height: 2em;
     color:#555753;
+    display:inline-block;
+
```

Comment:
Minor nitpick but you have a rogue newline here.

Reply:
hey @halkeye, I don't know how that newline has come up there. That is not showing on my local code. I think that can be ignored.

Reply:
its probably newline related, file is windows and you are mac or other way around.

run the file through cat and see what it looks like.

not a blocker to me

Reply:
oh alright, should I commit the code again in this case?

Reply:
Lets go with yes. Better to clean up after yourself. You can probably just edit the code directly in github.

If everyone leaves one or two random extra lines behind, it can get messy, better to stay consistent.

Reply:
Yes, I totally agree with that. I will clear the new line and edit it directly in the Github. Should I make a new PR or just edit it in this one only?

Reply:
You don't need a new PR. Its still related to this change. Edit your branch (ui or whatever) and it'll get picked up.

Reply:
Sounds great! I will do that